codelist creation.md

```diff
@@ -60,13 +60,11 @@ Once a draft codelist has been agreed, it must be signed-off by a "data expert"
 
 ## Add to [codelists.opensafely.org](https://codelists.opensafely.org)
 
-* Go to the openSAFELY [codelist admin page](https://codelists.opensafely.org/admin). You will need an editor account.
-* Under "CODELISTS", click "Add".
+* Go to the openSAFELY [new codelist page](https://codelists.opensafely.org/codelist/opensafely/). You will need an editor account.
 * Fill in the fields. Include lots of detail (specific guidance to follow).
-	* **Version**: this should be the current date.
 	* **CSV data**: _DO NOT_ copy the data from within spreadsheet software such as Excel (this can reformat or revalue cells, for example with very long numeric codes that are interpreted as integers). Instead, open the CSV file in a text editor such as notepad and copy the raw text data.
```

Comment:
We're no longer copying/pasting CSV data into a `textarea`, so this bit is out of date.  Instead, could you add a line about how to create a CSV file by exporting a spreadsheet from Excel?  I know that @brianmackenna has done this recently.

Reply:
So in excel there are two ways to change the file type from `xlsx` to `csv`. 

I recommend if you have newer versions of excel(see screenshot below)
1. Go to file
2. Go to "export" on the left handside ribbon
3. Select "change file type"

The second way is to
1. File
2. Save as
3. change the file type to .`csv`

This is the number way to do it on google but I have found this buggy. It makes a file that looks like a csv but when we have uploaded it to opencodelists it has broken. When its been investigated it turns out is not quite a csv. I have found the first way to work ok so far. 

![image](https://user-images.githubusercontent.com/40460354/90112253-a7737680-dd47-11ea-8271-a43b37a65bd5.png)

Reply:
Fixed in 155a981